### ISSUE TITLE
Error: onDrag called before onDragStart 

### ISSUE THREAD

**Issue by mlisthenewcool** [react-grid-layout/react-grid-layout#1454]
Sometimes `Error: onDrag called before onDragStart` is fired if you drop an item on top of another.

![Screenshot from 2021-04-18 14-36-58](https://user-images.githubusercontent.com/43213629/115145917-74ab1c00-a054-11eb-9789-30eb2d4c17ad.png)

**Comment by caowenhai888** [comment]
me too

**Comment by STRML** [comment]
Sorry, can't do much with this without a reproduction.

**Comment by pabloiea1995** [comment]
Having same issue, any updates?

**Comment by jehanjazz-1** [comment]
Same issue. 

I will try to reproduce it and drop the steps here once I find it

**Comment by amitpatil321** [comment]
In my case, I wanted to have a feature where user can maximize the div full screen, so I was applying a css fullscreen class. I was using `position: fixed !important` property and getting above error. 

I removed that property and now I don't see that error. Now I need to find a way to use position fixed and not let it throw error.

**Comment by FiadhMc** [comment]
I'm also having this issue. Same as above, I'm applying a fullscreen class with `position: fixed !important` on the div and then I see the error. Has anyone had any success fixing this? Or a workaround?

**Comment by FiadhMc** [comment]
Thanks so much for the reply! This worked perfectly; you probably saved me at least a few hours of trying to figure it out.

**Comment by amitpatil321** [comment]
Yeh I know, Because I have spent few days to figure out the way...Its almost year now, But when I saw you are in trouble, I remembered the hassle I faced and though to help you by sparing some time.

**Comment by FiadhMc** [comment]
Thanks so much, I had low expectations of getting any reply, never mind an actual solution!

**Comment by DRMPN** [comment]
I encountered an issue while using **Leaflet** and **RGL**. 

When I placed a map inside a container, the **Fullscreen button** stopped functioning for the entire screen and only worked within the local container. Despite trying various solutions, the problem persisted for several months with not a single error message in my console.

Interestingly, the issue was resolved by using the `useCSSTransforms={false}` prop!

A big thank you! ❤
